sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -283,60 +283,61 @@ void ng_ndp_nbr_adv_handle(kernel_pid_t iface, ng_pktsnip_t *pkt,
 
 void ng_ndp_retrans_nbr_sol(ng_ipv6_nc_t *nc_entry)
 {
-    if ((nc_entry->probes_remaining > 1) &&
-        ((ng_ipv6_nc_get_state(nc_entry) == NG_IPV6_NC_STATE_INCOMPLETE) ||
-         (ng_ipv6_nc_get_state(nc_entry) == NG_IPV6_NC_STATE_PROBE))) {
-        ng_ipv6_addr_t dst;
+    if ((ng_ipv6_nc_get_state(nc_entry) == NG_IPV6_NC_STATE_INCOMPLETE) ||
```

Comment:
Since this check is implementation dependent (regarding different ND for IPv6 and 6LoWPAN), I'd like to avoid that.